### PR TITLE
Release/1.0.1

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Mantis - 0902813
Fixes
- Title colour change
- Column title colour change
- Edit Picklist field persist error

### CODE REVIEW

File: widget/release_notes.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # What's New
-Minor bug fix:
+Minor bug fixes:
+    -Persists a change in 'Picklist' value.
+
```
Comment (cs-suyog-jinturkar) [inline widget/release_notes.md:5]: Please added "usability improvements"

File: widget/view.html [hunk 2]
```diff
@@ -1,8 +1,7 @@
-<div class="widget widget-container chart">
+<div class="widget widget-container ">
     <div class="display-flex-space-between margin-chart">
-        <div class="padding-right-0 padding-left-0 widget-dashboard-title-width"
-            data-ng-class="(page === 'dashboard' || page === 'reporting') ? 'widget-dashboard-title-width' : 'widget-title-width'">
-            <h5 class="margin-top-0 margin-bottom-0 text-overflow ng-binding">{{ config.title }}</h5>
+        <div class="padding-right-0 padding-left-0 widget-dashboard-title-width">
+            <h5 class="custom-label-size margin-bottom-0 section-border padding-left-md">{{ config.title }}</h5>
```
Comment (cs-suyog-jinturkar) [inline widget/view.html:4]: why this change?

Comment (cs-yash-patodkar) [reply widget/view.html:4]: For the colour
